src/openrct2/object/SceneryGroupObject.cpp

```diff
@@ -150,28 +150,28 @@ uint32_t SceneryGroupObject::ReadJsonEntertainerCostumes(const json_t* jCostumes
 uint32_t SceneryGroupObject::ParseEntertainerCostume(const std::string& s)
```

Comment:
Make this function return `EntertainerCostume` and just cast to `uint32_t` on the single place that calls it. Will become neater.

Reply:
Thanks, I've done so in the most recent commit.